hw/ip/csrng/data/csrng.hjson

```diff
@@ -207,6 +207,43 @@
         }
       ]
     },
+    { name: "INT_STATE_NUM",
+      desc: "Internal state number register",
+      swaccess: "rw",
+      hwaccess: "hro",
+      regwen: "REGEN",
+      fields: [
+        {
+            bits: "3:0",
+            name: "INT_STATE_NUM",
+            desc: '''
+                  Setting this field will set the number for which internal state can be
+                  selected for a read access. In normal operation, this register is hidden from
+                  firmware access. Only in life cycle mode set to "hw_debug" can the value
+                  of the internal state be read.
+                  '''
+        },
+      ]
+    },
+    {
+      name: "INT_STATE_VAL",
+      desc: "Internal state read access register",
+      swaccess: "ro",
+      hwaccess: "hrw",
+      hwext: "true",
+      hwre: "true",
+      fields: [
+        { bits: "31:0",
+          name: "INT_STATE_VAL",
+          desc: '''
+                Reading this register will dump out the contents of the selected internal state field.
+                Since the internal state is 384 bits wide, it will require 12 reads from this
+                register to see the entire field. Once 14 reads are completed, the read pointer
+                will be reset and reading to read another internal state.
```

Comment:
I'm not sure the end of this sentence makes sense. Do you mean that it goes on to the "next" field, or do you mean that we go back to the start of the current one?

Reply:
There are many errors here, so cleaned this up. The internal state number was wrong both here and in the doc section. They all match up now.